docker-compose.dist.yml

```diff
@@ -7,50 +7,31 @@ services:
         container_name: koillection
         build:
             dockerfile: Dockerfile.dev
-        restart: always
```

Comment:
Are you sure it should be removed? It works with this line as well.

Reply:
Yeah, it has been removed in the upstream. We will need to revisit all these when we sync with upstream.
If container exits with rc != 0 then restart the container.